constrained_ik/constrained_ik_constraint_plugin_description.xml

```diff
@@ -1,4 +1,4 @@
-<library path="lib/libconstrained_ik_constraints">
+<library path="libconstrained_ik_constraints">
```

Comment:
I changed this while trying to troubleshoot something, I didn't restore it back to how it was but it seems that catkin is smart enough to figure out where to find the lib.